src/zope/interface/tests/test_declarations.py

```diff
@@ -968,11 +961,14 @@ def test_w_non_descriptor_aware_metaclass(self):
         from zope.interface.interface import InterfaceClass
         IFoo = InterfaceClass("IFoo")
         class MetaClass(type):
-            def __getattribute__(self, name):
+            def __getattribute__(cls, name):
                 # Emulate metaclass whose base is not the type object.
                 if name == '__class__':
                     return self
```

Comment:
Heh. Nice catch! The test still works because `self` is defined in the outer scope.